sdk/framework/path.go

```diff
@@ -193,6 +193,9 @@ type DisplayAttributes struct {
 	// EditType is the optional type of form field needed for a property
 	// This is only necessary for a "textarea" or "file"
 	EditType string `json:"editType,omitempty"`
+
+	// HelpText is the optional helper text for display on the UI
+	HelpText string `json:"helpText,omitempty"`
```

Comment:
My only thoughts are around redundancy with `Description`. Do we anticipate that UI helptext is going to necessarily be different than the parameter description, which is what will appear with `path-help` and eventually in docs? Or could this be more of a style guide thing where Descriptions are written in a way that can be used in the UI cleanly?

Reply:
That is a great point, the goal of `HelpText` now (which, i realized might be a misnomer and perhaps `DefaultValueDescription` would be better) is to describe where the default value comes from. A style guide convention where that is always mentioned in the description sounds like a great option, but let me check with design to make sure! I can't imagine _more_ helpful information that comes with the description would be a bad thing